independent-publisher-connectors/AzureADApplications/apiDefinition.swagger.json

```diff
@@ -81,405 +123,519 @@
+				"type": "string",
+				"description": "The given name (first name) of the user (user type only)"
+			  },
+			  "mail": {
+				"type": "string",
+				"description": "The email of the user (user type only)"
+			  },
+			  "mobilePhone": {
+				"type": "string",
+				"description": "The mobile phone number of the user (user type only)"
+			  },
+			  "surname": {
+				"type": "string",
+				"description": "The user's surname (user type only)"
+			  },
+			  "userPrincipalName": {
+				"type": "string",
+				"description": "The user principal name (UPN) of the user  (user type only)"
+			  }
+			}
+		  }
+		}
+	  }
+	},
+	"Application_Definition": {
+		"type": "object",
+		"properties": {
+		  "id": {
+			"type": "string",
+			"description": "The unique identifier for the application"
+		  },
+		  "deletedDateTime": {
+			"type": "string",
+			"description": "The date and time the application was deleted. The DateTimeOffset type represents date and time information using ISO 8601 format and is always in UTC time."
+		  },
+		  "appId": {
+			"type": "string",
+			"description": "The unique identifier for the application that is assigned to an application by Azure AD"
+		  },
+		  "applicationTemplateId": {
+			"type": "string",
+			"description": "Unique identifier of the applicationTemplate"
+		  },
+		  "disabledByMicrosoftStatus": {
+			"type": "string",
+			"description": "Specifies whether Microsoft has disabled the registered application"
+		  },
+		  "createdDateTime": {
+			"type": "string",
+			"description": "The date and time the application was registered. The DateTimeOffset type represents date and time information using ISO 8601 format and is always in UTC time"
+		  },
+		  "displayName": {
+			"type": "string",
+			"description": "The display name for the application"
+		  },
+		  "description": {
+			"type": "string",
+			"description": "The description for the application"
+		  },
+		  "groupMembershipClaims": {
+			"type": "string",
+			"description": "The groups claim issued in a user or OAuth 2.0 access token that the application expects"
+		  },
+		  "identifierUris": {
+			"type": "array",
+			"items": {
+			  "type": "string"
+			},
+			"description": "The URIs that identify the application within its Azure AD tenant, or within a verified custom domain if the application is multi-tenant"
+		  },
+		  "isDeviceOnlyAuthSupported": {
+			"description": "Specifies whether this application supports device authentication without a user. The default is false"
+		  },
+		  "isFallbackPublicClient": {
+			"type": "boolean",
+			"description": "Specifies the fallback application type as public client, such as an installed application running on a mobile device. The default value is false which means the fallback application type is confidential client such as a web app"
+		  },
+		  "notes": {
+			"type": "string",
+			"description": "Notes relevant for the management of the application"
+		  },
+		  "optionalClaims": {

```

Comment:
@paulculmsee The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. can you please define the "type'. Or please confirm that no users will be affected by this change.

Reply:
Hello @paulculmsee,

this property used to have a type of string. please see below
![image](https://user-images.githubusercontent.com/97551789/218852272-99000e94-4132-4302-aeca-dcba43df4013.png)

In this update there is no type defined.